roles/transformers/tasks/main.yml

```diff
@@ -29,8 +28,8 @@
     checksum: "sha1:{{ lookup('url', pkg_url + '.sha1') }}"
     dest: "{{ download_location }}/{{ pkg_name }}"
     mode: "0644"
-    timeout: 570
   loop: "{{  imagemagick_packages }}"
+  when: ansible_distribution != 'Rocky'
```

Comment:
We should avoid spreading tasks that essentially do the same thing in different places.
Let's use distro specific vars to provide a  common interface to a single task